src/ApparentHorizons/StrahlkorperGr.cpp

```diff
@@ -90,36 +90,33 @@ template <typename Frame>
 tnsr::ii<DataVector, 3, Frame> extrinsic_curvature(
     const tnsr::ii<DataVector, 3, Frame>& grad_normal,
     const tnsr::i<DataVector, 3, Frame>& unit_normal_one_form,
-    const tnsr::II<DataVector, 3, Frame>& upper_spatial_metric) noexcept {
-  // Form projector
-  tnsr::Ij<DataVector, 3, Frame> projector(get<0>(unit_normal_one_form).size(),
-                                           0.0);
-  const auto& unit_normal_vector =
-      raise_or_lower_index(unit_normal_one_form, upper_spatial_metric);
+    const tnsr::I<DataVector, 3, Frame>& unit_normal_vector) noexcept {
+  // Project symmetrized gradient
+  auto extrinsic_curvature(grad_normal);
+
+  Scalar<DataVector> nInJgradnij(get<0, 0>(grad_normal).size(), 0.0);
```

Comment:
I personally would have an easier time parsing this variable name if it were `nI_nJ_gradnij`